packages/ui/src/components/editor/panels/Scenes/modals/RenameScene.tsx

```diff
@@ -62,6 +62,7 @@ export default function RenameSceneModal({ sceneName, refetch, scene }: Props) {
       className="w-[50vw] max-w-2xl"
       onSubmit={handleSubmit}
       onClose={PopoverState.hidePopupover}
+      submitButtonDisabled={newSceneName.value === sceneName}
```

Comment:
let's also validate the name for disabling submit
![Screenshot from 2024-07-23 12-32-00](https://github.com/user-attachments/assets/127c4580-efb0-4981-827a-3751a17e4722)

Reply:
implemented
![image](https://github.com/user-attachments/assets/db4918ae-5674-4430-9dbf-513bb7683246)